molecule/default/playbook.yml

```diff
@@ -16,24 +16,23 @@
             - LANG
             - LANGUAGE
             - LC_*
-          PrintMotd: no
-          Subsystem: sftp  /usr/lib/openssh/sftp-server
```

Comment:
Did you remove this intentionally?

Reply:
Yes, for 8.2 subdir based config this is defined in the `/etc/ssh/sshd_config`, and as such is redundant. I didn't want to create a separate scenario for pre and post 8.2 configs.